nfs-ganesha-trigger/config/definitions/nfs-ganesha-trigger.yml

```diff
@@ -0,0 +1,62 @@
+- job:
+    name: nfs-ganesha-trigger
+    node: master
+    project-type: freestyle
+    defaults: global
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    properties:
+      - github:
+          url: https://github.com/nfs-ganesha/nfs-ganesha
+    discard-old-builds: true
+    logrotate:
+      daysToKeep: 1
+      numToKeep: 10
+      artifactDaysToKeep: -1
+      artifactNumToKeep: -1
+
+    triggers:
+      - github
+
+    scm:
+      - git:
+          url: https://github.com/nfs-ganesha/nfs-ganesha
+          browser: auto
+          # The default is to build and publish every branch.
+          # Uncomment this for testing:
+          branches:
+            - 'origin/master'
+            - 'origin/next'
+          skip-tag: true
+          timeout: 20
+          wipe-workspace: true
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph
+          browser: auto
+          # The default is to build and publish every branch.
+          # Uncomment this for testing:
+          branches:
+            - 'origin/master'
+            - 'origin/jewel'
+            - 'origin/kraken'
+          skip-tag: true
+          # set timeout to 2 hours after ceph jobs get triggered
+          timeout: 7200
```

Comment:
``timeout`` is used to define how long a ``git`` operation can run before being timed out. I'm not sure that's how you're intending to use this here.